### PR TITLE
Fast-Lane-Reservation: Vehicles

### PR DESCRIPTION
**Pull request by degisew**
### Fast-Lane-Reservation: Vehicles
**In this PR, I implemented the ff features:**
- Created restful routes
- Created Vehicle controller
- Created Vehicle models

### CODE REVIEW

**Review by Maxwell011 — APPROVED** [review]
There are some changes that needs to be done

File: app/controllers/vehicles_controller.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  # Use callbacks to share common setup or constraints between actions.
+  def set_vehicle
+    @vehicle = Vehicle.find(params[:id])
+  end
+
+  # Only allow a list of trusted parameters through.
+  def vehicle_params
+    params.fetch(:vehicle, {})
+  end
+end
```
Comment (kennyegun24) [inline app/controllers/vehicles_controller.rb:50]: Hello @degisew, you did not follow best practice for creating APIs by putting your controller in the app/controllers/api/v1 directory.

**Review by kennyegun24 — COMMENTED** [review]
### @degisew, I have dropped some reviews of what you should change

**Review by Maxwell011 — CHANGES_REQUESTED** [review]
There are some changes

**Pull request merged**